### PR TITLE
Fix JsonHubProtocolOptions doc comment

### PR DESCRIPTION
**Pull request by halter73**

### CODE REVIEW

File: src/SignalR/common/Protocols.Json/src/JsonHubProtocolOptions.cs [hunk 1]
```diff
@@ -7,7 +7,7 @@
 namespace Microsoft.AspNetCore.SignalR
 {
     /// <summary>
-    /// Options used to configure a <see cref="JsonHubProtocolOptions"/> instance.
+    /// Options used to configure a <see cref="JsonSerializerOptions"/> instance.
```
Comment (BrennanConroy) [inline src/SignalR/common/Protocols.Json/src/JsonHubProtocolOptions.cs]:
```suggestion
    /// Options used to configure a <see cref="JsonHubProtocol"/> instance.
```

Comment (halter73) [reply src/SignalR/common/Protocols.Json/src/JsonHubProtocolOptions.cs]: Or would the following be better?

```suggestion
    /// Options used to configure the <see cref="JsonSerializerOptions"/> used by SignalR's <see cref="JsonHubProtocol"/> instance.
```

Comment (BrennanConroy) [reply src/SignalR/common/Protocols.Json/src/JsonHubProtocolOptions.cs]: That's ok, my main concern is that these options can be for more than just the Serializer, like maybe we'll allow different protocol versions via these options in the future.

**Comment by halter73** [comment]
@BrennanConroy I went with your initial recommendation.

**Review by BrennanConroy — APPROVED** [review]

**Pull request merged**